### PR TITLE
fix: Make Speaker description width 100% for 992px to 1200px screen

### LINKED ISSUE #2322: Speaker description width is not 100%
**Actual Behaviour**
.speaker-description(Class) width is not the full width of display or 100%

**Expected Behaviour**
width will 100% of class .speaker-description

**Steps to reproduce it**
1. http://sched.eventyay.com/
2. Click on FOSSASIA Summit 2018
3. Then change width 50% to 100%
@media (max-width: 992px)
.speaker-with-bio .speaker-description {
    text-align: center;
    width: 100%;
}

Before here width is 50% which is not the full width of the screen that's why the text is not in the center

**Screenshots of the issue**
WhatsApp Image 2020-07-13 at 10 57 54 AM

**Would you like to work on the issue?**
Yes, I would like to work on this issue.

### PR DESCRIPTION
**Pull request by dsnehasish74**
Previously it was of width 50% by default ............ according to this issue I made it 100%.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-
-

Fixes #2322

### CODE REVIEW

**Review by digitalPlayer1125 — COMMENTED** [review]
This seems fine. 
Maybe add padding-top as well so that the author name doesn't stick to the image?

**Review by digitalPlayer1125 — CHANGES_REQUESTED** [review]
Just a change, I think having padding-top as 20px would be better than 40px, as 40px is more of overuse of whitespace. Compare the two screenshots as attached. Makes sense?
image
image

**Review by digitalPlayer1125 — APPROVED** [review]
Great!

**Review by Raunaksingh100 — COMMENTED** [review]
Plz fixed your travis cl failed ........

**Comment by digitalPlayer1125** [comment]
@Raunaksingh100 As you can see that it is just a change in CSS, the travis CI error has nothing to do with it [It is caused due to Selenium Timeout]

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -1396,8 +1396,9 @@ a {
     }
 
     .speaker-description {
+      padding-top:40px;
```
[the remark this answers is not in the document]
Comment (Raunaksingh100) [reply src/backend/_scss/application.scss]: > Colon after property should be followed by one space

@digital read this

[the remark this answers is not in the document]
Comment (digitalPlayer1125) [reply src/backend/_scss/application.scss]: This has been already resolved.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**